src/types/amplitude/v8.ts

```diff
@@ -16,6 +20,11 @@ export interface CurrencyId_Stellar {
     value: Asset
 }
 
+export interface CurrencyId_ZenlinkLPToken {
```

Comment:
I manually added this because, for some reason, it is not generated from the archive using sqd typegen.

Reply:
For Amplitude I think our ZenlinkLPToken change is just not live yet.